### PR TITLE
RHDEVDOCS-2085 remove unnecessary pics from serverless

### PR DESCRIPTION
**Pull request by ghost**
https://issues.redhat.com/browse/RHDEVDOCS-2085

- [x] Reviewed by <QE/SME>
- [x] peer-reviewed by DT docs 
- [x] peer-reviewed by ocp docs
- merge to 4.6

### CODE REVIEW

**Comment by pmacko1** [comment]
pics from topology view are here
https://docs.openshift.com/container-platform/4.5/serverless/knative_serving/splitting-traffic-between-revisions.html
and here
https://docs.openshift.com/container-platform/4.5/serverless/event_sources/serverless-apiserversource.html

**Comment by abrennan89** [comment]
@pmacko1 when you are deleting images maybe do a grep search to make sure they're not used anywhere else, then delete the actual image file also if it only appears in the module you're removing it from.
Should be fine to remove the image files in this case since they're specific to this module. Thanks.

File: modules/odc-creating-apiserversource.adoc [hunk 1]
```diff
@@ -9,20 +9,11 @@
...
-+
-image::apiserversource-odc.png[Create an ApiServerSource]
 . Configure the **ApiServerSource** settings:
 .. Enter `v1` as the **APIVERSION**, and `Event` as the **KIND**.
 // .. Select **Resource** as the **Mode**. **Mode** is the mode that the receive adapter controller runs in. `Ref` sends only the reference to the resource. `Resource` sends the full resource.
 // TODO: clarify what this is used for. Out of scope for this PR since not required.
 .. Select the **Service Account Name** for the service account that you created.
-+
-image::apiserver-odc-config-1.png[Select the Service Account Name and Sink]
-.. Select the targeted Knative service from the dropdown menu in **Sink** -> **Knative Service**.
```
Comment (abrennan89) [inline modules/odc-creating-apiserversource.adoc:21]: It looks like this step also is deleted by mistake maybe, re selecting the Sink?

Comment (pmacko1) [reply modules/odc-creating-apiserversource.adoc:21]: yeah idk why i deleted this

File: modules/odc-creating-apiserversource.adoc [hunk 2]
```diff
@@ -9,20 +9,12 @@
 
 . Navigate to the **Add** page and select **Event Source**.
 . In the **Event Sources** page, select **ApiServerSource** in the **Type** section.
-+
-image::apiserversource-odc.png[Create an ApiServerSource]
```
Comment (pmacko1) [inline modules/odc-creating-apiserversource.adoc:13]: is not elsewhere in docs + deleted it

File: modules/odc-creating-apiserversource.adoc [hunk 3]
```diff
@@ -9,20 +9,12 @@
...
 . Configure the **ApiServerSource** settings:
 .. Enter `v1` as the **APIVERSION**, and `Event` as the **KIND**.
 // .. Select **Resource** as the **Mode**. **Mode** is the mode that the receive adapter controller runs in. `Ref` sends only the reference to the resource. `Resource` sends the full resource.
 // TODO: clarify what this is used for. Out of scope for this PR since not required.
 .. Select the **Service Account Name** for the service account that you created.
-+
-image::apiserver-odc-config-1.png[Select the Service Account Name and Sink]
```
Comment (pmacko1) [inline modules/odc-creating-apiserversource.adoc:20]: is not elsewhere in docs + deleted it

File: modules/odc-creating-apiserversource.adoc [hunk 4]
```diff
@@ -9,20 +9,12 @@
...
 .. Select the targeted Knative service from the dropdown menu in **Sink** -> **Knative Service**.
-// .. Add details about the image:
-// General -> Application Name, Name
-// image::apiserver-odc-config-2.png[Configuring the ApiServerSource]
```
Comment (pmacko1) [inline modules/odc-creating-apiserversource.adoc:24]: is not elsewhere in docs + deleted it

**Comment by ahardin-rh** [comment]
LGTM! Looks like there are some conflicts and the commits require a squash, but these removals look good!

**Comment by pmacko1** [comment]
@ahardin-rh OK conflicts resolved, didnt have my local branches updated. @abrennan89 could you please review?

**Comment by pmacko1** [comment]
@invincibleJai to be consistent, I'd ask you for a review as well ;)

**Comment by invincibleJai** [comment]
@pmacko1 will there be separate PR to add new screenshots or these are not needed?

**Comment by pmacko1** [comment]
these are not needed @invincibleJai

**Comment by invincibleJai** [comment]
Thanks , Looks good!!

**Comment by abrennan89** [comment]
LGTM, thanks!

**Pull request merged**